superset-frontend/plugins/legacy-plugin-chart-sunburst/src/Sunburst.js

```diff
@@ -381,15 +382,17 @@ function Sunburst(element, props) {
       .append('text')
       .attr('class', 'path-abs-percent')
       .attr('y', yOffsets[offsetIndex])
-      .text(`${absolutePercString} of total`);
+      // eslint-disable-next-line prefer-template
+      .text(absolutePercString + ' ' + t('of total'));
 
     if (conditionalPercString) {
       offsetIndex += 1;
       gMiddleText
         .append('text')
         .attr('class', 'path-cond-percent')
         .attr('y', yOffsets[offsetIndex])
-        .text(`${conditionalPercString} of parent`);
+        // eslint-disable-next-line prefer-template
+        .text(conditionalPercString + ' ' + t('of parent'));
```

Comment:
Same here to abide by the lint rules rather than disabling them, if I'm not mistaken:
```suggestion
        // eslint-disable-next-line prefer-template
        .text(`${conditionalPercString} ${t('of parent')}`);
```